### PR TITLE
[docs] Update deploy-bare-metal-multi-cluster.md

### PR DESCRIPTION
**Pull request by fengtao1998**
Revise official documents to make them easy to understand and use
Update deploy-bare-metal-multi-cluster.md

### CODE REVIEW

**Comment by fengtao1998** [comment]
@Jennifer88huang PTAL

File: site2/docs/deploy-bare-metal-multi-cluster.md [hunk 1]
```diff
@@ -127,15 +124,15 @@ $ bin/pulsar-daemon start zookeeper
...
 
 #### Single-cluster Pulsar instance
 
 If your Pulsar instance consists of just one cluster, then you can deploy a configuration store on the same machines as the local ZooKeeper quorum but run on different TCP ports.
 
-To deploy a ZooKeeper configuration store in a single-cluster instance, add the same ZooKeeper servers that the local quorom uses to the configuration file in [`conf/global_zookeeper.conf`](reference-configuration.md#configuration-store) using the same method for [local ZooKeeper](#local-zookeeper), but make sure to use a different port (2181 is the default for ZooKeeper). The following is an example that uses port 2184 for a three-node ZooKeeper cluster:
+To deploy a ZooKeeper configuration store in a single-cluster instance, add the same ZooKeeper servers that the local quorom uses to the configuration file in [`conf/global_zookeeper.conf`](reference-configuration.md#configuration-store) using the same method for local ZooKeeper](#local-zookeeper), but make sure to use a different port (2181 is the default for ZooKeeper). The following is an example that uses port 2184 for a three-node ZooKeeper cluster:
```
Comment (Jennifer88huang) [inline site2/docs/deploy-bare-metal-multi-cluster.md:135]: 1. The first sentence is too long, try to use shorter sentences, less than 25 words for a full sentence.
2. the link for "local ZooKeeper](#local-zookeeper)" is not correct, try to fix it.

**Review by Jennifer88huang — COMMENTED** [review]
Generally, good job~

**Comment by Jennifer88huang** [comment]
@sijia-w you can also help review, thank you.

File: site2/docs/deploy-bare-metal-multi-cluster.md [hunk 2]
```diff
@@ -86,8 +77,8 @@ Directory | Contains
 
 Each Pulsar instance relies on two separate ZooKeeper quorums.
 
-* [Local ZooKeeper](#deploy-local-zookeeper) operates at the cluster level and provides cluster-specific configuration management and coordination. Each Pulsar cluster needs to have a dedicated ZooKeeper cluster.
-* [Configuration Store](#deploy-the-configuration-store) operates at the instance level and provides configuration management for the entire system (and thus across clusters). An independent cluster of machines or the same machines that local ZooKeeper uses can provide the configuration store quorum.
+* Local ZooKeeper operates at the cluster level and provides cluster-specific configuration management and coordination. Each Pulsar cluster needs to have a dedicated ZooKeeper cluster.
```
Comment (Jennifer88huang) [inline site2/docs/deploy-bare-metal-multi-cluster.md:80]:
```suggestion
* Local ZooKeeper operates at the cluster level and provides cluster-specific configuration management and coordination. Each Pulsar cluster needs a dedicated ZooKeeper cluster.
```

File: site2/docs/deploy-bare-metal-multi-cluster.md (hunk 1 above)
Comment (fengtao1998) [reply site2/docs/deploy-bare-metal-multi-cluster.md]: I'm very sorry. I fixed it yesterday, but it seems that the save failed. I'll modify this file again. I will @you when I finish it.

**Comment by fengtao1998** [comment]
@Jennifer88huang  PTAL

**Review by Jennifer88huang — APPROVED** [review]

**Pull request merged**